public/wp-config.php

```diff
@@ -82,33 +82,20 @@
 	require dirname( dirname( __FILE__ ) ) . '/wp-config-local.php';
 }
 
-if ( ! defined( 'FORCE_SSL_ADMIN' ) ) {
-	define( 'FORCE_SSL_ADMIN', true );
-}
-
-/**
- * For developers: WordPress debugging mode.
- *
- * Change this to true to enable the display of notices during development.
- * It is strongly recommended that plugin and theme developers use WP_DEBUG
- * in their development environments.
- *
- * For information on other constants that can be used for debugging,
- * visit the Codex.
- *
- * @link https://codex.wordpress.org/Debugging_in_WordPress
- */
-if ( ! defined( 'WP_DEBUG' ) ) {
-	define( 'WP_DEBUG', true );
-}
-
-if ( ! defined( 'WP_DEBUG_LOG' ) ) {
-	define( 'WP_DEBUG_LOG', true );
-}
-
-if ( ! defined( 'WP_DISABLE_FATAL_ERROR_HANDLER' ) ) {
-	define( 'WP_DISABLE_FATAL_ERROR_HANDLER', true );
+// Define constants that haven't been overridden by wp-tests-config-local.php.
+$constants = array(
+	'FORCE_SSL_ADMIN'                => true,
+	'WP_DEBUG'                       => true,
+	'WP_DEBUG_LOG'                   => true,
+	'WP_DISABLE_FATAL_ERROR_HANDLER' => true,
+	'JETPACK_DEV_DEBUG'              => true,
+);
+foreach ( $constants as $constant_name => $constant_value ) {
+	if ( ! defined( $constant_name ) ) {
+		define( $constant_name, $constant_value );
+	}
 }
+unset( $constants );
```

Comment:
`$constant_name` and `$constant_value` should be unset as well, to be safe - I'm not entirely sure about `$constant_name`, but `$constant_value` will otherwise definitely be available as a global.

Reply:
Good point